modello-maven-plugin/src/main/java/org/codehaus/modello/maven/AbstractModelloGeneratorMojo.java

```diff
@@ -160,11 +173,29 @@ public void execute() throws MojoExecutionException {
 
         parameters.setProperty(ModelloParameterConstants.PACKAGE_WITH_VERSION, Boolean.toString(packageWithVersion));
 
-        if (packagedVersions.size() > 0) {
+        if (!packagedVersions.isEmpty()) {
             parameters.setProperty(
                     ModelloParameterConstants.ALL_VERSIONS, StringUtils.join(packagedVersions.iterator(), ","));
         }
 
+        if (licenseText != null && !licenseText.trim().isEmpty()) {
+            String license = "";
+            try {
+                Path licenseFile = Paths.get(licenseText);
+                if (Files.isRegularFile(licenseFile)) {
+                    license = String.join("\n", Files.readAllLines(licenseFile));
```

Comment:
Should we managed in special way for line-end char?
Should be default or system specific?

Reply:
I just copied from other (existing) code that adds "generated" header. Unsure why would it need to differ.